solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -708,7 +856,7 @@ describe("RandomBeacon - Relay", () => {
   async function assertMembersEligible(members: number[], groupSize: number) {
     for (let i = 0; i < members.length; i++) {
       expect(
-        // eslint-disable-next-line no-await-in-loop,@typescript-eslint/no-unused-expressions
+        // eslint-disable-next-line no-await-in-loop
```

Comment:
I think it can be moved at the top of the file do disable for all for-loops

Reply:
https://github.com/keep-network/keep-core/pull/2671/commits/2291bf7a4f9e8c987eee300f1d9302c243d790bc